pkg/storagecapabilities/storagecapabilities.go

```diff
@@ -289,11 +289,39 @@ func createDellUnityCapabilities() []StorageCapabilities {
 	}
 }
 
-func createDellPowerCapabilities() []StorageCapabilities {
+func createDellPowerMaxCapabilities() []StorageCapabilities {
 	return []StorageCapabilities{
 		{rwx, block},
```

Comment:
@akalenyu according to the Dell doc PowerMax/PoweFlwex/PowerStore suppport RWX/ROX for Filesystem volume mode **with NFS only**.  Shouldn't we add them to the capabilities?

Reply:
Yeah, there's a good chance this driver needs us to differentiate over one of storage class parameters,
for example, I see FibreChannel/ISCSI/NFS examples in here:
https://github.com/dell/csi-powermax/tree/main/samples/storageclass
Which could mean that somebody using powermax-nfs will fail to create block volumes altogether https://github.com/kubevirt/containerized-data-importer/blob/842342452f5028098453d43f7aa4bb4ec0d67e67/pkg/storagecapabilities/storagecapabilities.go#L292-L294 But I don't see anything obvious to key off in the storage class examples.

TBH usually it's best to have the storage provider make the recommendation

Reply:
So the way they indicate this is nfs is via `csi.storage.k8s.io/fstype`?
This is a little surprising since this key is normally used as a configurable to specify which filesystem you want created on top of the block device (ext4/xfs/...) For example - https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/docs/parameters.md

Reply:
After checking I think you are right. I can't find anything else that mentions `nfs` or `protocol` in the code in our context (except [this](https://github.com/dell/csi-powerflex/blob/7b8d376691aef258233c69f2afcab41f9974ef1a/service/node.go#L151) but I don't think that's what we're looking for). Only DELL unity mentions the protocol explicitly as in the example [here](https://github.com/dell/csi-unity/blob/1f42af327f4130df65c5532f6029559e4ab579b5/samples/storageclass/unity-nfs.yaml#L50), but all the others do not.
Maybe we'll just ask a contact from DELL what the string is, if we have one?